torrt/base_bot.py

```diff
@@ -0,0 +1,28 @@
+from utils import WithSettings, BotObjectsRegistry
+
+
+class BaseBot(WithSettings):
```

Comment:
Maybe drop ``bot`` in favor of ``actor`` (as in #28) or something else more generic to describe subsystem built to reach out and communicate with remote systems, e.g. messengers? Becase only ``bot`` seems too narrow.
How do you think?

Reply:
I think we can do it later when all became clear with #28

Reply:
#28 here is just for illustration.

Unfortunatelly we won't be able to change anything after the release (otherwise we break the backward compatibility — both API and CLI), and would be obliged to use e.g. ``configure_bot`` (instead of, say ``configure_actor``) forever even its'possible that it'd be semantically incorrect to name other integrations as ``bots``. We certainly could initiate some deprectation processes but that'd be painful both for us and for end-users. That's why I propose a more generic name.

Reply:
It is easy to do, you know. But seems for usual user it is hard to understand what is actor. Generally even I can't imagine another way to communicate with actor. I know that it can appears in the future and we should be ready for it. So I can rename bots to actors if you want. But I need help with documentation in this case :)

Reply:
Ok, lt's settle with ``bot``.